lmdeploy/pytorch/modeling/modeling_baichuan.py

```diff
@@ -33,7 +33,7 @@
 
 from .configuration_baichuan import BaiChuanConfig
 
-logger = get_logger(__name__)
+logger = get_logger('lmdeploy')
```

Comment:
can we set the default value of name to `'lmdeploy'` instead of `None` for convenience in here?
https://github.com/InternLM/lmdeploy/blob/fa07c5e97a4bb07c3cbfe77822f4c38974cfc409/lmdeploy/utils.py#L10